### PR TITLE
Fix 159 QA bug findings v 30 round four

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket
Fix bugs

# Description

# Actions
Add button and delete links

### CODE REVIEW

File: src/core/context/ThemeContext.tsx [hunk 1]
```diff
@@ -2,13 +2,15 @@ import { createContext, ReactNode, useContext } from 'react';
 import { CssBaseline, ThemeProvider as MuiThemeProvider, useMediaQuery } from '@mui/material';
 import lightTheme from '../../../styles/theme/light';
 import darkTheme from '../../../styles/theme/dark';
-import useLocalStorage from '../hooks/useLocalStorage';
+// import useLocalStorage from '../hooks/useLocalStorage';
```
Comment (YasielCabrera) [inline src/core/context/ThemeContext.tsx:5]: Is this comment needed?

File: src/core/context/ThemeContext.tsx [hunk 2]
```diff
@@ -22,13 +24,14 @@ const useThemeContext = () => useContext(ThemeContext);
...
-  const isDarkOS = useMediaQuery(DARK_SCHEME_QUERY);
+  const userSystemThemePreferenceDark = useMediaQuery(DARK_SCHEME_QUERY);
 
-  const [themeMode, setThemeMode] = useLocalStorage<ThemeMode>(
-    'themeMode',
-    isDarkOS && cookies.darkMode === 'true' ? 'dark' : 'light'
-  );
+  // console.log('userSystemThemePreferenceDark', userSystemThemePreferenceDark);
+  const defaultInitialTheme = userSystemThemePreferenceDark && cookies.darkMode === 'true' ? 'dark' : 'light';
+  // console.log({ defaultInitialTheme });
```
Comment (YasielCabrera) [inline src/core/context/ThemeContext.tsx:31]: Are those comments needed?

File: src/core/hooks/useLocalStorage.ts [hunk 3]
```diff
@@ -1,55 +1,57 @@
-import { Dispatch, SetStateAction, useEffect, useState } from 'react';
+import { useCallback, useEffect, useState } from 'react';
 import { useCookies } from 'react-cookie';
-import useEventListener from './useEventListener';
 
-type SetValue<T> = Dispatch<SetStateAction<T>>;
-
-function useLocalStorage<T>(key: string, initialValue: T): [T, SetValue<T>] {
```
Comment (YasielCabrera) [inline src/core/hooks/useLocalStorage.ts:7]: Could be used https://github.com/DSpotDevelopers/makerdao-ses-dashboard/blob/master/src/core/utils/local-storage.ts in order to simplify the hook a bit?

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: src/core/context/ThemeContext.tsx (hunk 2 above)
Comment (alejandrocabriales) [reply src/core/context/ThemeContext.tsx:31]: I will delete

File: src/core/context/ThemeContext.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply src/core/context/ThemeContext.tsx:5]: I will delete

File: src/core/hooks/useLocalStorage.ts (hunk 3 above)
Comment (alejandrocabriales) [reply src/core/hooks/useLocalStorage.ts:7]: This will not need any more, this file is delete in the next PR, but I will check that code for use in the PR

**Review by YasielCabrera — APPROVED** [review]

**Pull request merged**